docs/conf.py

```diff
@@ -1,11 +1,22 @@
+comments_config = {
+    "hypothesis": True,
+    "dokieli": True,
+    "utterances": {
+        "repo": "github-org/github-repo",
+        "optional": "config",
+    },
```

Comment:
Utterances doesn't work as nice in my experience. Just remove it, we only need hypothesis for #37